tests/Http/Helpers.phpt

```diff
@@ -14,11 +14,11 @@ require __DIR__ . '/../bootstrap.php';
 test(function() {
 	Assert::true( Helpers::ipMatch('192.168.68.233', '192.168.68.233') );
 	Assert::false( Helpers::ipMatch('192.168.68.234', '192.168.68.233') );
-	Assert::true( Helpers::ipMatch('192.168.64.0', '192.168.68.233/12') );
```

Comment:
No, the test was fine even though it tested something different than you intended.